src/std/fs.ab

```diff
@@ -76,3 +78,9 @@ pub fun change_owner(user: Text, path: Text): Bool {
 
     return false
 }
+
+/// Finds all files or directories matching a file glob.
+pub fun glob(path: Text): [Text] {
+    let files = unsafe $eval "for file in {path}; do test -e \\\"\\\$file\\\" && echo \\\"\\\$file\\\"; done"$
```

Comment:
`test -e X` is basically `[ -e X ]`, right? wouldn't it be better to use that in terms of readibility?

Reply:
also i dont understand why do you need `eval` here. it just evaluates the string as bash, right? why not just evaluate it directly via amber?

Reply:
I personally find `test` more readable, but I don't mind changing it.

Reply:
Normally I would never use `eval` due to injection attack concerns, but in this case it's necessary because none of the other combinations work with globs containing escaped spaces.  Please see my post on https://github.com/amber-lang/amber/issues/463 for explanation.